frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/request-vehicles/simulated-region-overview-behavior-request-vehicles.component.html

```diff
@@ -2,17 +2,9 @@
     <h5>Anfragenziel</h5>
 
     <ng-container *ngIf="{ time: nextTimeoutIn$ | async } as nextTimeoutIn">
-        <span *ngIf="(waitingForAnswer$ | async) === true" class="text-muted"
-            >Wartet auf die Antwort einer Anfrage von</span
-        >
-        <span
-            *ngIf="
-                nextTimeoutIn.time !== undefined &&
-                (waitingForAnswer$ | async) === false
-            "
-            class="text-muted"
+        <span *ngIf="nextTimeoutIn.time !== undefined" class="text-muted"
             >Nächste Anfrage potentiell in
```

Comment:
I would put here "nächste potenzielle Anfrage in" because "nächste Anfrage potenziell in "  implies that there is a request that potentially is going to be send and the point is potentiall rather than the existance of the request

Reply:
ea1e25dc6137aa070d3436b384aa4e6f865d1372